bittensor/constants.py

```diff
@@ -15,7 +15,29 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+# Standard Library
+import asyncio
+from typing import Dict, Type
+
+# 3rd Party
+import aiohttp
+
 
 ALLOWED_DELTA = 4_000_000_000  # Delta of 4 seconds for nonce validation
 V_7_2_0 = 7002000
 NANOSECONDS_IN_SECOND = 1_000_000_000
+
+#### Dendrite ####
+DENDRITE_ERROR_MAPPING: Dict[Type[Exception], tuple] = {
```

Comment:
Why do we store a constant in an external `bittensor/constants.py` module if it is only used in the `bittensor/dendrite.py` module? Let's store it in the same module where it's using.

Reply:
this is subjective and shouldn't be a blocking comment.